packages/math-input/src/components/keypad/mobile-keypad.tsx

```diff
@@ -221,10 +223,12 @@ const styles = StyleSheet.create({
         transitionProperty: "transform",
         transform: "translate3d(0, 100%, 0)",
     },
-
     activeKeypadContainer: {
         transform: "translate3d(0, 0, 0)",
     },
+    inactiveKeypadContainer: {
+        display: "none",
```

Comment:
I don't think this is going to work will it? This would destroy the animation right?

Reply:
Matthew was completely right about my original approach destroying the animation! Not sure why it worked the first few times I tested. I looked into "aria-hidden" and it looks like we don't want to be using it on focusable elements, and it doesn't pass the property along to child focusable elements. 

Instead I've opted to use "visibility" within our css transition, so that we get the best of all worlds. This allows the animations to persist and ensures that the keypad isn't visible to screen readers while closed.